src/ts/localization/i18n.ts

```diff
@@ -0,0 +1,178 @@
+import vocabularyDe from './languages/de.json';
+import vocabularyEn from './languages/en.json';
+import { LocalizationConfig } from '../uimanager.js';
+
+export const defaultTranslations: BitmovinPlayerUiTranslations = {
+  'en': vocabularyEn,
+  'de': vocabularyDe,
+};
+
+const defaultLocalizationConfig: LocalizationConfig = {
+  language: 'en',
+  browserLanguageDetection: true,
+  translations: defaultTranslations,
+};
+
+type LocalizableCallback = () => string;
```

Comment:
This is not a callback, because it does not "call back" to anywhere. It is simply a function that creates an output. It is also not a "localizable" because we cannot execute any action on it to localize it, i.e. it does not have a `setLocale` or equivalent. Please rename to `Localizer` - in the sense that this is a function that does emit localized output, and the action it does it to localize a string - so it "localizes" and is therefore a "localizer".

Please also rename the method that generates it from `getLocalizableCallback` to `getLocalizer`.